test/unit/fact_cleaner_test.rb

```diff
@@ -0,0 +1,38 @@
+require 'test_helper'
+
+class FactCleanerTest < ActiveSupport::TestCase
+  let(:cleaner) do
```

Comment:
Believe me I'd prefer all the goodness of spec-like syntax too, but I was pushed back several times :-) I'm happy we now use `let` and `describe` but rest usually does not slip in.